<launch>
  <!-- 启动思岚c1雷达 -->
  <node pkg="rplidar_ros"  type="rplidarNode" name="rplidarNode" output="screen">
    <!-- <remap from="scan" to="scan"/> -->
    <param name="serial_port"         type="string" value="/dev/ttyUSB0"/>
    <param name="serial_baudrate"     type="int"    value="460800"/>
    <!-- 雷达坐标系 -->
    <param name="frame_id"            type="string" value="Lidar_link"/>
    <param name="inverted"            type="bool"   value="false"/>
    <param name="angle_compensate"    type="bool"   value="true"/>
    <param name="scan_frequency"      type="double" value="10.0"/>
    <param name="scan_mode"           type="string" value="Standard"/>
  </node>
</launch>